Tests/Event/MessageListEventTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the AdminLTE bundle.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use KevinPapst\AdminLTEBundle\Event\MessageListEvent;
+use KevinPapst\AdminLTEBundle\Model\MessageModel;
+use KevinPapst\AdminLTEBundle\Model\UserModel;
+use PHPUnit\Framework\TestCase;
+
+class MessageListEventTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function total_should_be_zero_and_max_null_when_there_are_no_messages()
+    {
+        $event = new MessageListEvent();
+        $this->assertEquals(0, $event->getTotal());
+        $this->assertEquals(null, $event->getMax());
+    }
+
+    /**
+     * @test
+     */
+    public function total_should_be_equal_the_number_of_messages_if_max_is_greater_then_the_number_of_message()
+    {
+        $event = new MessageListEvent(10);
+        $messages = $this->generateNbMessages(7);
+
+        foreach ($messages as $message) {
+            $event->addMessage($message);
+        }
+
+        $this->assertEquals(7, $event->getTotal());
+        $this->assertEquals(10, $event->getMax());
+        $this->assertEquals(7, count($event->getMessages()));
+    }
+
+    /**
+     * @test
+     */
+    public function total_should_be_equal_the_number_of_messages_and_count_message_should_equal_max_when_max_is_lower_then_the_number_of_messages()
```

Comment:
These function names hurt my 👀 hehehe
Did you run the codestyle fixer? I thought there would be a check in place to allow camelCase only.
But nevermind, its a test, so it doesn't matter...